Artifacts/windows-chocolatey/install-choco-package.ps1

```diff
@@ -89,7 +95,16 @@ function Install-Packages
     )

```

Comment:
Pass `$AllowEmptyChecksums` and `$IgnoreChecksums` as parameters. You're currently referencing the globally defined ones.

Reply:
Won't do.